code/modules/vehicles/interior/interactable/vendors.dm

```diff
@@ -0,0 +1,383 @@
+		list("Autoinjector (Dexalin+)", round(scale * 3), /obj/item/reagent_container/hypospray/autoinjector/dexalinp, VENDOR_ITEM_REGULAR),
+		list("Autoinjector (Epinephrine)", round(scale * 3), /obj/item/reagent_container/hypospray/autoinjector/adrenaline, VENDOR_ITEM_REGULAR),
+		list("Autoinjector (Inaprovaline)", round(scale * 3), /obj/item/reagent_container/hypospray/autoinjector/inaprovaline, VENDOR_ITEM_REGULAR),
+		list("Autoinjector (Kelotane)", round(scale * 3), /obj/item/reagent_container/hypospray/autoinjector/kelotane, VENDOR_ITEM_REGULAR),
+		list("Autoinjector (Oxycodone)", round(scale * 3), /obj/item/reagent_container/hypospray/autoinjector/oxycodone, VENDOR_ITEM_REGULAR),
+		list("Autoinjector (QuickClot)", round(scale * 3), /obj/item/reagent_container/hypospray/autoinjector/quickclot, VENDOR_ITEM_REGULAR),
+		list("Autoinjector (Tramadol)", round(scale * 3), /obj/item/reagent_container/hypospray/autoinjector/tramadol, VENDOR_ITEM_REGULAR),
+		list("Autoinjector (Tricord)", round(scale * 3), /obj/item/reagent_container/hypospray/autoinjector/tricord, VENDOR_ITEM_REGULAR),
+
+		list("MEDICAL UTILITIES", -1, null, null),
+		list("Surgical Line", round(scale * 2), /obj/item/tool/surgery/surgical_line, VENDOR_ITEM_REGULAR),
+		list("Synth-Graft", round(scale * 2), /obj/item/tool/surgery/synthgraft, VENDOR_ITEM_REGULAR),
+		list("Health Analyzer", round(scale * 4), /obj/item/device/healthanalyzer, VENDOR_ITEM_REGULAR),
+		list("Stasis Bag", round(scale * 6), /obj/item/bodybag/cryobag, VENDOR_ITEM_REGULAR),
+		list("Syringe", round(scale * 3), /obj/item/reagent_container/syringe, VENDOR_ITEM_REGULAR)
+	)
+
+//MED APC version of Blood Dispenser
+/obj/structure/machinery/cm_vending/sorted/medical/blood/vehicle
+	name = "\improper MM Blood Dispenser"
+	desc = "Marine Med brand Blood Pack Dispensery for vehicles."
+	icon = 'icons/obj/vehicles/interiors/general.dmi'
+
+	req_access = list()
+
+//----------------------SUPPLY VENDORS--------------------
+
+//Combined vehicle version of req guns and ammo vendors. Starts, basically, empty, but can become a resupply point if marines bother to stock it with ammo and weapons.
+/obj/structure/machinery/cm_vending/sorted/vehicle_supply
+	name = "\improper ColMarTech Automated Supply Vendor"
+	desc = "An automated supply rack hooked up to a vehicle storage of various firearms, explosives and ammunition types. Used for storing and transporting supplies between FOB and frontline."
+	icon = 'icons/obj/vehicles/interiors/general.dmi'
+	icon_state = "supply"
+	req_access = null
+	vendor_theme = VENDOR_THEME_USCM
+
+	unacidable = TRUE
+	unslashable = TRUE
+	indestructible = TRUE
+	wrenchable = FALSE
+	hackable = FALSE
+	density = FALSE
+
+/obj/structure/machinery/cm_vending/sorted/vehicle_supply/Initialize()
+	. = ..()
+	GLOB.cm_vending_vendors += src
+
+/obj/structure/machinery/cm_vending/sorted/vehicle_supply/Destroy()
+	GLOB.cm_vending_vendors -= src
+	return ..()
```

Comment:
we should really get around to having a better way to do this. for another pr maybe?